packages/node_modules/@webex/internal-plugin-devices/src/index.js

```diff
@@ -0,0 +1,11 @@
+import {registerInternalPlugin} from '@webex/webex-core';
+
+import Devices from './devices';
+import config from './config';
+
+registerInternalPlugin('devices', Devices, {
+  config
+});
+
+export {default} from './devices';
+export {default as Devices} from './devices';
```

Comment:
Duplicate export?

Reply:
This is how it was set up in the `internal-plugin-wdm`, I think these logically do the same thing, but it may also allow:
```js
import {Devices} from '@webex/internal-plugin-devices';
import Devices from '@webex/internal-plugin-devices';
```

Reply:
Since this is a new plugin, keep it as one and use the syntax used in other plugins
`export {default} from './devices`

Reply:
Pushed and done!